music21/analysis/neoRiemannian.py

```diff
@@ -204,15 +212,40 @@ def isNeoR(c1, c2):
     pc1 = P(c1)
     rc1 = R(c1)
 
+    UFM = chromaticMediants(c1, transformation='UFM')
+    USM = chromaticMediants(c1, transformation='USM')
+    LFM = chromaticMediants(c1, transformation='LFM')
+    LSM = chromaticMediants(c1, transformation='LSM')
+
+    Slide = S(c1)
+    Neben = N(c1)
```

Comment:
we are now making nine different chords, but in the case of L, only one will be used.  Can we make it a loop so that there is only one `if x.normalOrder == c2.normalOrder` test?  Pylint is going to complain that the number of if statements in the loop exceeds the maximum.  Note also that computing `c2.normalOrder` adds run time but it will always be the same for each comparison.  Cache it once.